common/http/src/main/java/io/helidon/common/http/Http1HeadersParser.java

```diff
@@ -58,6 +67,10 @@ public static WritableHeaders<?> readHeaders(DataReader reader, int maxHeadersSi
 
             Http.HeaderName header = readHeaderName(reader, maxLength, validate);
             maxLength -= header.defaultCase().length() + 2;
+            // Skip spaces after header name
+            while (' ' == reader.lookup()) {
```

Comment:
`java.lang.Character.isWhitespace()` would be a better choice here

Reply:
This was reverted to main branch code.